packages/gatsby-plugin-mdx/loaders/mdx-loader.js

```diff
@@ -120,11 +121,15 @@ module.exports = async function (content) {
 
   let mdxNode
   try {
-    mdxNode = await createMDXNode({
+    // This node attempts to break the chicken-egg problem, where parsing mdx
+    // allows for custom plugins, which can receive a mdx node
+    ;({ mdxNode } = await createMDXNode({
```

Comment:
Seems like an errant `;`?

Reply:
That's Prettier because due to the object destructuring pattern `{x} = y` has to be wrapped in parens, for otherwise it becomes a block. And Prettier will prefix a semi-colon to any statement starting with parenthesis, to prevent an accidental call expression because we don't use semi-colons.

Reply:
Actually, https://github.com/gatsbyjs/gatsby/pull/25790 will fix this as well (after rebasing) because then it'll be 
```suggestion
    let { mdxNode } = await createMDXNode({
```

Reply:
Nice